composer.json

```
@@ -4,7 +4,7 @@
     "description": "Basic API to handle HTTP redirects with the Flow Framework",
     "license": "MIT",
     "require": {
-        "neos/flow": ">=4.0 <6.0 || dev-master",
+        "neos/flow": "^6.0",
```

Allowing master makes it easier to install in the dev distribution

```suggestion
        "neos/flow": "^6.0 || dev-master",
```

---

I personally don't like the "|| dev-master” approach. Current dev-master will be 6.0 and after that the package may not be compatible with the next master. I recommend to use `"neos/flow-development-collection": "dev-master as 6.0"` and only allow that version her as it is compatible now.A benefit is, that the package can the be released without changing the version constraint.

---

Also fine for me. Didn't know that worked 👍